index.js

```diff
@@ -7,13 +7,34 @@ const through = require('through2')
 
 module.exports = JSONToCsv
 
-function JSONToCsv () {
-  return pumpify(JSONStream.parse(), split(), CsvStream())
+const csvOpts = ['separator', 'newline', 'headers', 'sendHeaders']
```

Comment:
Right, that's fine, path is fine for the json path too since it just takes a string arg. But json is good too.